docs/how_to/keys/grok.md

```diff
@@ -0,0 +1,106 @@
+# How to get and use an xAI Grok API key
+
+Grok is a family of Large Language Models (LLMs) developed by xAI. You can access these models via the GrokPrompt config node. To make use of these, however, you'll need to have an XAI account, and generate an API key. It's worth noting as well, that xAI is a paid service, and to make use of it, you'll need to set up billing details on their website.
```

Comment:
little comment - it's `xAI`. In one of these you've got `XAI`. Fix that, and good to go!